File: app/src/main/java/app/fit/fitndflow/data/dto/categories/ExerciseDto.java
package app.fit.fitndflow.data.dto.categories;

import com.google.gson.annotations.SerializedName;

import java.io.Serializable;

import app.fit.fitndflow.data.dto.StringInLanguages;


public class ExerciseDto implements Serializable {

    @SerializedName("id")
    public int id;
    @SerializedName("name")
    public StringInLanguages exerciseName;

    @SerializedName("primaryCategory")
    public int primaryCategory;

}

File: app/src/main/java/app/fit/fitndflow/data/dto/categories/ExerciseDto.kt
package app.fit.fitndflow.data.dto.categories

import app.fit.fitndflow.data.dto.StringInLanguagesDto
import com.google.gson.annotations.SerializedName
import java.io.Serializable

data class ExerciseDto(@SerializedName("id") var id: Int?, @SerializedName("name") var exerciseName: StringInLanguagesDto?): Serializable
